README.md

```diff
@@ -178,6 +178,9 @@ The docker-compose-standalone.yml file is meant for local testing/troubleshootin
 To utilize this, pass the -f flag to the docker-compose command as follows:
 > docker-compose -f docker-compose-confluent-cloud.yml up
 
+### Data & Config Files
```

Comment:
Let's expand this description a little bit to include that we expect the drive be mounted to the /ppm_data directory. Maybe we could even include and example for setting it up in a BSM configuration, what files we'd expect to find in that directory and how we'd convey that to the application via the environment variables.

Reply:
This section has been expanded upon. Let me know if you want anything else in there.